go-controller/pkg/ovn/policy.go

```diff
@@ -449,31 +442,24 @@ func (oc *Controller) updateACLLoggingForPolicy(np *networkPolicy, logLevel stri
 	if !np.created {
 		return NetworkPolicyNotCreated
 	}
-
-	acls := oc.buildNetworkPolicyACLs(np, logLevel)
-	ops, err := libovsdbops.UpdateACLsLoggingOps(oc.nbClient, nil, acls...)
-	if err != nil {
-		return err
+	// Predicate for given network policy ACLs
+	p := func(item *nbdb.ACL) bool {
+		return item.ExternalIDs[namespaceACLExtIdKey] == np.namespace && item.ExternalIDs[policyACLExtIdKey] == np.name
```

Comment:
I would probably have this and what we do in `buildACLAllow` in a shared method `buildPolicyCommonExternalIds` to make sure that we avoid changin one thing without changing the other.

Reply:
I hope to get there with couple more PRs, don't think it belongs here

Reply:
I kinda does for me but I am fine if you are working in follow-up PRs that will handle that.